### PR TITLE
remove CPU & RAM measurements from syncthing GUI

### PR DESCRIPTION
**Pull request by jesselucas**
### Purpose

Remove CPU & RAM measurements from GUI to address issue #6249

### Testing

Test UI in browser

### Screenshots

Screen Shot 2020-03-03 at 3 54 36 PM

## Authorship

Your name and email will be added automatically to the AUTHORS file
based on the commit metadata.

### CODE REVIEW

**Comment by calmh** [comment]
I think we should also remove the CPU measurement code from the backend and report that stat as zero in the API. Whether that constitutes and API change that should trigger a 1.5 release depends on how loud people yell... I personally don't care much. We should add a note to the relevant REST API docs though documenting that this happened.

The RAM usage stuff goes into the usage report so should be retained as-is I think, and so can stay in the API as well...

File: man/syncthing-faq.7 [hunk 1]
```diff
@@ -128,12 +128,6 @@ firewall\-setup to enable direct connections.
 Second, if one of the devices is a very low powered machine (a Raspberry Pi,
 or a phone, or a NAS, or similar) you are likely constrained by the CPU on
 that device. See the next question for reasons Syncthing likes a faster CPU.
-You can verify this by looking at the CPU utilization in the GUI. If it is
```
Comment (calmh) [inline man/syncthing-faq.7]: The man pages are auto generated from the syncthing/docs repo, so the change should happen there instead.

File: lib/api/api.go [hunk 2]
```diff
@@ -81,7 +81,6 @@ type service struct {
 	fss                  model.FolderSummaryService
 	urService            *ur.Service
 	systemConfigMut      sync.Mutex // serializes posts to /rest/system/config
-	cpu                  Rater
```
Comment (calmh) [inline lib/api/api.go]: Is this Rater interface used anywhere else? Otherwise kill it

Comment (jesselucas) [reply lib/api/api.go]: Good catch!

**Pull request merged**